lesson1/dev/sass/_banner.sass

```diff
@@ -0,0 +1,36 @@
+#banner 
+  width: 100%
+  height: 400px
+  position: relative
+  overflow: hidden
+  ol 
+    position: absolute
+    z-index: 200
+    bottom: 10px
+    left: 50%
+    transform: translateX(-50%)
+    li 
+      width: 16px 
+      height: 16px
+      border: 2px solid #fff 
+      border-radius: 100%
+      display: inline-block
+      margin: 0 5px
+      cursor: pointer
+      &.active 
+        background: #fff
+        border: transparent
+  .slider
+    position: absolute
+    top: 0
+    left: 0
+    width: 300%
+    height: 100%
+    display: flex
```

Comment:
`display: flex`
Tham khảo 
`-webkit-flex: 1; /* Safari 6.1+ */
    -ms-flex: 1; /* IE 10 */ 
    flex: 1;
`
Em lên tạo mixin cho những phần này , sửa toàn bộ các phần khác cũng như vậy